server/ws/src/server.ts

```diff
@@ -316,7 +327,7 @@ class TSessionManager implements SessionManager {
     workspaceName: string
   ): Promise<Pipeline> {
     if (LOGGING_ENABLED) {
```

Comment:
maybe move LOGGING_ENABLED inside ctx so we don't have to wrap every call with `if`?

Reply:
nope, it mostly for tests to check failed cases